### PR TITLE
Update useremail.html.erb 

### LINKED ISSUE #6962: Include Emails of Banned Users in Moderator Email Search
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Moderators have access to a feature that allows them to lookup profiles by email.  During a recent attempt to unban a mistakenly banned user (#6883) it was discovered that this feature does not include the emails of banned users.

In discussion with the moderation team, there are some good use cases for including these emails:

1) if a user created their account a long time ago, they may not remember there username (or remember it correctly).  In those instances, finding a profile by email would be the only recourse.  

2) Including banned users may eliminate a round of back and forth if the person emailing the moderation team is using the email address they signed up with.

The current solution (related #7109)is to create a check box that when toggled will add banned from 2015 onward to the search.  We also need to add some text explaining what the check box does and the impact that using it may have on the site.

**NOTE:** _See the **update** section below for a link to the post in this conversation where the steps are located to complete the FTO._

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** 
The steps needed to complete the FTO can be found in the conversation below at [this post](https://github.com/publiclab/plots2/issues/6962#issuecomment-570727398).

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by shipcy**
Fixes #6962  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

File: app/views/admin/useremail.html.erb [hunk 1]
```diff
@@ -1,27 +1,24 @@
...
+   <h3>User lookup by email:</h3> 
 
-  <p>Moderators and admins have the ability to search for usernames associated with a given email address. This ability is meant to assist in answering requests submitted by email where the user forgets to specify their username; clearly the email address of the sender is known.</p>
-  <%= form_tag do %>
-    <%= text_field_tag(:address, value = @address) %>
-    <%= button_tag(:search, type: 'submit', class: "btn btn-primary") do %>
-      <i class="fa fa-search fa fa-white"></i>
-    <% end %>
-</button>
-  <% end %>
```
Comment (nstjean) [inline app/views/admin/useremail.html.erb]: So here the form has been removed and replaced with the new checkbox. We're going to have to put it back in! This is what this section should look like:
```
   <p>Moderators and admins have the ability to search for usernames associated with a given email address. This ability is meant to assist in answering requests submitted by email where the user forgets to specify their username; clearly the email address of the sender is known.</p>
   <%= form_tag do %>
      <%= text_field_tag(:address, value = @address) %>
      <%= button_tag(:search, type: 'submit', class: "btn btn-primary") do %>
         <i class="fa fa-search fa fa-white"></i>
      <% end %>
      </button>
      <label>
         <input type="checkbox" id="myCheck" name="include_banned" />
         Include banned accounts after 2015 (careful: can result in server slowness for all users)
      </label>
  <% end %>
```

**Comment by nstjean** [comment]
Also at the top of this pull request can you edit and add in the number of the original issue? This creates a link so we know what this belongs to!

**Comment by nstjean** [comment]
This just needs to be changed to 6962
FireShot Capture 184 - Update useremail html erb by shipcy · Pull Request #7137 · publiclab__ - github com

**Comment by shipcy** [comment]
> This just needs to be changed to 6962
> FireShot Capture 184 - Update useremail html erb by shipcy · Pull Request #7137 · publiclab__ - github com

@nstjean done.

File: app/views/admin/useremail.html.erb [hunk 2]
```diff
@@ -1,27 +1,31 @@
...
+   <p>Moderators and admins have the ability to search for usernames associated with a given email address. This ability is meant to assist in answering requests submitted by email where the user forgets to specify their username; clearly the email address of the sender is known.</p>
+   <%= form_tag do %>
+       <%= text_field_tag(:address, value = @address) %>
+       <%= button_tag(:search, type: 'submit', class: "btn btn-primary") do %>
+          <i class="fa fa-search fa fa-white"></i>
+       <% end %>
+       </button>
+       <label>
+         <input type="checkbox" id="myCheck" name="include_banned" />
+         Include banned accounts after 2015 (careful: can result in server slowness for all users)
```
Comment (jywarren) [inline app/views/admin/useremail.html.erb]: This part looks good!

**Comment by jywarren** [comment]
Awesome, thanks so much!!! 🎉

**Comment by skilfullycurled** [comment]
@shipcy, you did it!!  

This one was a bit more involved than other FTO’s and you made so much progress independently.

If you still feel shakey in terms of knowing what you’re doing, don’t worry, I did too. The support isn’t a one time deal, we’re here to help you as you get more and more confident.

If you want to contribute some more (and I hope you will!) if you don’t find an issue that feels like the right difficulty for where you’re at, there are lots of larger projects/initiatives that people are working on and you can leave a comment asking if they need help (that’s sort of what I did).  People can then craft tasks for you to contribute that meet your experience level as you gain more confidence!

Again, congratulations, and great job!

**Comment by skilfullycurled** [comment]
Oh, also, @shipcy, I’ll grab a screenshot because your work is a moderator function only.

**Comment by skilfullycurled** [comment]
Bringing in at @ebarry and @bronwen9 since they’re the ones whose experience moderating inspired this feature!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by skilfullycurled** [comment]
Actually, @shipcy, if you want more than a screenshot of how it looks, but would like to see the feature in action, make an account on publiclab.org (if you haven’t already), let me know here. Then I’ll ban you (which I need to do to show you how it works) use the feature, take a screenshot, and of course, unban you.  : )

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by shipcy** [comment]
> @shipcy, you did it!!
> 
> This one was a bit more involved than other FTO’s and you made so much progress independently.
> 
> If you still feel shakey in terms of knowing what you’re doing, don’t worry, I did too. The support isn’t a one time deal, we’re here to help you as you get more and more confident.
> 
> If you want to contribute some more (and I hope you will!) if you don’t find an issue that feels like the right difficulty for where you’re at, there are lots of larger projects/initiatives that people are working on and you can leave a comment asking if they need help (that’s sort of what I did). People can then craft tasks for you to contribute that meet your experience level as you gain more confidence!
> 
> Again, congratulations, and great job!

@skilfullycurled It was impossible to do this job without your help. Thank you so much! And yes I wanna contribute a lot to this org.

**Comment by shipcy** [comment]
> Oh, also, @shipcy, I’ll grab a screenshot because your work is a moderator function only.

@skilfullycurled Can you tell me which screenshot are you talking about here?

**Comment by shipcy** [comment]
> Actually, @shipcy, if you want more than a screenshot of how it looks, but would like to see the feature in action, make an account on publiclab.org (if you haven’t already), let me know here. Then I’ll ban you (which I need to do to show you how it works) use the feature, take a screenshot, and of course, unban you. : )
> 
> Sent with [GitHawk](http://githawk.com)

@skilfullycurled I have created my account on publiclab.org. What's next?

**Comment by skilfullycurled** [comment]
@shipcy, I want you to be able to see the feature you implemented on the website.  But in this case, it’s a function that only people with higher levels of access can see, so I’ll have to log in to take a screenshot.  Normally, you’d get the experience of trying it out yourself.  So, it’s not as satisfying but at least you’ll see it!

**Comment by skilfullycurled** [comment]
@shipcy, I forgot to ask for your username.  Let me know that.

After that, you don’t have to do anything else.  I’ll go ahead and ban you, take the screenshot and then unban you.

Of course, if you want the fullest experience, I suppose you could do something that would get you banned such as creating a research note insulting me.

: P

**Comment by shipcy** [comment]
@skilfullycurled I have signed up that through GitHub only, so my username will remain the same @shipcy.

**Comment by shipcy** [comment]
> @shipcy, I forgot to ask for your username. Let me know that.
> 
> After that, you don’t have to do anything else. I’ll go ahead and ban you, take the screenshot and then unban you.
> 
> Of course, if you want the fullest experience, I suppose you could do something that would get you banned such as creating a research note insulting me.
> 
>  
@skilfullycurled LOL, no! I won't create any research note insulating you.

**Comment by shipcy** [comment]
> @skilfullycurled I have signed up that through GitHub only, so my username will remain the same @shipcy.

Sorry @skilfullycurled! My username is @cshipra23.

**Comment by skilfullycurled** [comment]
> @skilfullycurled LOL, no! I won't create any research note insulating you.

I want everyone to have as authentic an experience as possible.  : P

> Sorry @skilfullycurled! My username is @cshipra23.

No apologies necessary!

**Comment by shipcy** [comment]
> > @skilfullycurled LOL, no! I won't create any research note insulating you.
> 
> I want everyone to have as authentic an experience as possible. : P

But you are the one who helped me a lot in my first Pull request.

**Pull request merged**